test/lexers/javascript-lexer.test.js

```diff
@@ -91,16 +92,6 @@ describe('JavascriptLexer', () => {
     Lexer.extract(content)
     done()
   })
-
-  it('supports the acorn-es7 plugin', (done) => {
```

Comment:
I wouldn't remove this test, that's the point of leaving it as a `devDependency`, so we can run a test and check it works. I would move it in the `describe` block above though.